samples/Mailchimp.gs

```diff
@@ -0,0 +1,100 @@
+/*
+ * This sample demonstrates how to configure the library for the Mailchimp API.
+ * Instructions on how to generate OAuth credentuals is available here:
+ * https://mailchimp.com/developer/guides/how-to-use-oauth2/#Step_1%3A_Register_your_application
+ */
+
+var CLIENT_ID = '...';
+var CLIENT_SECRET = '...';
+var OAUTH_HOST = 'login.mailchimp.com';
```

Comment:
After scanning the docs, it seems like the OAuth host is always this value. If there is no need for it to change then I'd prefer the variable be removed and the value just be baked into the URLs below.

Reply:
That’s a good suggestion.